airbyte-ci/connectors/auto_merge/README.md

```diff
@@ -0,0 +1,33 @@
+# `auto_merge`
```

Comment:
💡 There could be a section on "Purpose" e.g. explain why we have automerge and that the goal is to merge connectors with as little manual airbyte intervention as possible